lms/djangoapps/certificates/signals.py

```diff
@@ -31,7 +31,7 @@
 from student.models import CourseEnrollment
 
 log = logging.getLogger(__name__)
-CERTIFICATE_DELAY_SECONDS = 2
+CERTIFICATE_DELAY_SECONDS = 0.5
```

Comment:
```suggestion
CERTIFICATE_DELAY_SECONDS = getattr(settings, 'CERTIFICATE_DELAY_SECONDS', 2)
```

then in server-vars.yml we need to set:

```yml
EDXAPP_ENV_EXTRA:
  CERTIFICATE_DELAY_SECONDS: 0.5
```

Reply:
@bryanlandia I have no idea about auto-cert generation to be honest. It seems like a good idea, but I haven't reviewed it to know about multi-tenancy.

We're focusing on the big upcoming release, but on the flip side anything course-related is usually multi-tenant friendly.